tests/test_images.py

```diff
@@ -1,21 +1,25 @@
 import pytest
+import uuid
 from io import BytesIO
 from aiodocker import utils
 from aiodocker.exceptions import DockerError
 
 
+def _random_name():
+    return "aiodocker-" + uuid.uuid4().hex[:7]
+
+
 @pytest.mark.asyncio
 async def test_build_from_remote_file(docker):
     remote = ("https://raw.githubusercontent.com/aio-libs/"
               "aiodocker/master/tests/docker/Dockerfile")
 
-    tag = "image:1.0"
+    tag = "%s:1.0" % _random_name()
     params = {'tag': tag, 'remote': remote, 'stream': True}
     stream = await docker.images.build(**params)
 
-    async for output in stream:
-        if "Successfully tagged image:1.0\n" in output:
-            break
+    async for _ in stream:  # noqa: F841
```

Comment:
Now the funny part is that the previous code has problems too. I first tried to use it and catch from the output and assert on the output. But I realized that the "output" is no string, it's a dict, and my assert failed. The actual log line is in `output['stream']`.

So I updated the code and then the test failed because depending on the version of Docker, you may get an upper case, a lower case here and there, etc...

So in the end I felt like the only interesting thing to do here is to wait for the build to complete because this for loop doesn't actually do anything else than that.